.travis.yml

```diff
@@ -24,7 +24,7 @@ matrix:
 
 install:
   - pip install -q --use-mirrors nose python-dateutil $NUMPY pep8==1.5.7 pyparsing pillow
-  - sudo apt-get update && sudo apt-get -qq install inkscape libav-tools mencoder
+  - sudo apt-get update && sudo apt-get -qq install inkscape libav-tools mencoder dvipng texlive-latex-base texlive-latex-extra texlive-fonts-recommended
```

Comment:
I did that --no-install-recommends for building the documentation. As I remember it it saved a few 100 MB download. In any case this is only relevant on the colour overhaul branch. On master we are using a container build which handles this different.